src/actions/modular-pipelines.js

```diff
@@ -2,8 +2,13 @@ export const TOGGLE_MODULAR_PIPELINE_ACTIVE = 'TOGGLE_MODULAR_PIPELINE_ACTIVE';
 
 /**
  * Toggle a modular pipeline item's highlighting on/off (or array of modular pipelines)
+<<<<<<< HEAD
+ * @param {String|Array} modularPipelineIDs Modular pipeline id(s)
+ * @param {Boolean} active True if modualr pipeline(s) active
+=======
```

Comment:
```suggestion
 * @param {String|Array} modularPipelineIDs Modular pipeline id(s)
 * @param {Boolean} active True if modualr pipeline(s) active
```

You can then remove lines 9, 10, and 11 below.